scripts/config/msu/string.nut

```diff
@@ -27,4 +27,24 @@
 		}
 		return true;
 	}
+
+	function color( _string, _color )
```

Comment:
the argument order should take _color first and then _string as it will look cleaner in code with longer strings.